src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/GitController.kt

```diff
@@ -18,4 +18,22 @@ class GitController(context: ActionContext,
         val response = orderlyServerAPI.get("/git/commits?branch=${branch}", context)
         return passThroughResponse(response)
     }
+
+    fun status(): String
+    {
+        val response = orderlyServerAPI.get("/v1/reports/git/status/", context)
```

Comment:
I don't know why codcov is saying the the status endpoint isn't being covered because the test is right here.